packages/ui/origin-ui-v2/libs/ui/validation/src/yup/setYupLanguage.ts

```diff
@@ -0,0 +1,61 @@
+import { setLocale } from 'yup';
+import printValue from 'yup/lib/util/printValue';
+import { TFunction } from 'i18next';
+
+export const setYupLanguage = (t: TFunction) => {
+  const translate = (path: string) => {
+    return (values: any) => t(`validations.yup.${path}`, values);
+  };
+  setLocale({
```

Comment:
I am not sure if we will be using yup-specific localization, as long as we are using general localization